action.yml

```diff
@@ -22,6 +22,18 @@ inputs:
     description: 'What flake8 version to use, if none is installed (default is latest)'
     required: false
     default: ''
+  plugins:
+    description: 'List of plugins to install before running, This is passed directly to pip install'
+    required: false
+    default: ''
+  error_classes:
+    description: 'List of flake8 error classes to classify as Error.'
```

Comment:
Either please fix all descriptions end with a dot, or remove the dots here. Consistency :D